services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> None:
+
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+
+    if session_date:
+        return movie_session.filter(show_time__date=session_date)
+
+    return movie_session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+    session_id: int,
+    show_time: datetime = None,
+    movie_id: int = None,
+    cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.filter(id=session_id)
+
+    if show_time is not None:
+        movie_session.update(show_time=show_time)
+    if movie_id is not None:
+        movie_session.update(movie_id=movie_id)
+    if cinema_hall_id is not None:
+        movie_session.update(cinema_hall_id=cinema_hall_id)
```

Comment:
No need to make 3 queries, instead you can get object set values and then save or form update data and just once use update method